app/styles/_site-chrome.scss

```diff
@@ -6,6 +6,22 @@
   /* So the side nav will position correctly */
 }
 
+/* Hide certain elements based on window size. This must be done in css
+since */
```

Comment:
missing the end of this comment

Reply:
idk about you but i want as much suspense and mystery in my comments as possible